tests/e2e/proxy_key_updates_test.go

```diff
@@ -63,27 +85,17 @@ func TestProxyKeyUpdating(t *testing.T) {
 		{
 			name: "Given I remove the ProxyKeys access to the environment",
 			args: args{
-				org:               org2,
-				project:           project3Identifier,
-				editProxyKeyScope: removeEnvironment,
+				org:                org2,
+				project:            project3Identifier,
+				envIdentifier:      testhelpers.GetSecondaryEnvironment(),
+				proxyKeyIdentifier: proxyKeyIdentifier,
+				editProxyKeyScope:  editScope,
 			},
 			expected: expected{
-				statusCode:        http.StatusOK,
+				statusCode:        http.StatusUnauthorized,
 				numFeatureConfigs: 0,
 			},
 		},
-		{
-			name: "Given I re-add the ProxyKeys access to the environment",
-			args: args{
-				org:               org2,
-				project:           project3Identifier,
-				editProxyKeyScope: addEnvironment,
-			},
-			expected: expected{
-				statusCode:        http.StatusOK,
-				numFeatureConfigs: 2,
-			},
```

Comment:
This last test is redundant now that the test resets the config here https://github.com/harness/ff-proxy/pull/244/files#diff-1c25e4fe9cd84c6698098ef2fdcdab77ca7e669a6cd75f136b8bcaa0a7e4ed20R36